lib/languages.ts

```diff
@@ -192,16 +216,18 @@ const definitions: Record<LanguageKey, LanguageDefinition> = {
         logoUrlDark: null,
         formatter: null,
         previewFilter: null,
+        monacoDisassembly: null,
     },
-    cppx_cppfront: {
-        name: 'Cppx-cppfront',
-        monaco: 'cppx-cppfront',
+    cpp2_cppfront: {
+        name: 'Cpp2-cppfront',
+        monaco: 'cpp2-cppfront',
         extensions: ['.cpp2'],
```

Comment:
shouldn't this be .cpp, or does that give issues?

Reply:
Herb wanted it called "Cpp2" vs "Cpp" vs "Cppx".